BrandonBeltran/hackerrank/03/JavaHashset.java

```diff
@@ -0,0 +1,24 @@
+package bootcamp;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+/* This solution is the one I came on my own without Hackerrank's code.
+*  To be honest, I like it better, I include it for the sake of completion*/
+
+public class JavaHashset {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        int sets = Integer.parseInt(scanner.nextLine());
+
+        Set<String> names = new HashSet();
```

Comment:
Best practice in Collections or Map Variable Name, is to use singular name and them interface name: numberList, change it in all variables